docs/data/base/components/portal/portal.md

```diff
@@ -8,18 +8,28 @@ packageName: '@mui/base'
 
 # Portal
 
-<p class="description">The portal component renders its children into a new "subtree" outside of the current DOM hierarchy.</p>
+<p class="description">The <code>Portal</code> component lets you render its children into a DOM node that exists outside of its own DOM hierarchy.</p>
 
-The children of the portal component will be appended to the `container` specified.
-The component is used internally by the [`Modal`](/material-ui/react-modal/) and [`Popper`](/material-ui/react-popper/) components.
+`Portal` is a utility component built around [React's `createPortal()` API](https://reactjs.org/docs/portals.html).
+It gives you the functionality of `createPortal()` in a convenient component form.
 
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+:::info
+💡 According to [the React docs](https://reactjs.org/docs/portals.html), portals are useful when "you need the child element to visually 'break out' of its container."
+:::
 
-## Example
+The `Portal` component is used internally by Material UI's [`Modal`](material-ui/react-modal/) and [`Popper`](material-ui/react-popper/) components.
```

Comment:
At this point it's:

- The URL is broken 404
- MUI in general?
- Keep the user in MUI Base?

```suggestion
The `Portal` component is used internally by MUI's [`Modal`](/base/react-modal/) and [`Popper`](/base/react-popper/) components.
``` @